7thWrapperLib/Wrap.cs

```diff
@@ -488,11 +488,11 @@ public enum EMoveMethod : uint {
 
 
             bool result = Win32.WriteFile(hFile, lpBuffer, nNumberOfBytesToWrite, out lpNumberOfBytesWritten, ref lpOverlapped);
-            System.Diagnostics.Debug.WriteLine(String.Format("Write {0} bytes on {1}", lpNumberOfBytesWritten, hFile.ToInt32()));
+            //System.Diagnostics.Debug.WriteLine(String.Format("Write {0} bytes on {1}", lpNumberOfBytesWritten, hFile.ToInt32()));
 
             if (_saveFiles.ContainsKey(hFile)) {
                 int offset = SetFilePointer(hFile, 0, IntPtr.Zero, EMoveMethod.Current);
-                System.Diagnostics.Debug.WriteLine(String.Format("Write {0} bytes to {1} at offset {2}", lpNumberOfBytesWritten, _saveFiles[hFile], offset));
+                //System.Diagnostics.Debug.WriteLine(String.Format("Write {0} bytes to {1} at offset {2}", lpNumberOfBytesWritten, _saveFiles[hFile], offset));
```

Comment:
I commented those lines because they were bringing a lot of noise in TurboLog, and also it slows down a lot the game while having Turbolog opened. In case they are needed just let me know, but honestly, as we dump failures now, it's not really required to highlight what is being patched.